HIP/hip-20.md

```diff
@@ -0,0 +1,75 @@
+- hip: 20
+- title: Standardized URI scheme to reference Hedera data
+- author: Khoa Luong khoa.luong@luthersystems.com, Sam Wood sam.wood@luthersystems.com,
+Stephanie Yi stephanie.yi@luthersystems.com, Declan Fox declan.fox@luthersystems.com
+- type: Standards Track
+- category: Service
+- status: Draft
+- created: 2021-06-29
+- discussions-to:
```

Comment:
Could you create a new discussion thread here: https://github.com/hashgraph/hedera-improvement-proposal/discussions and link back to this new discussion here?